shared/js/ui/templates/tracker-networks.es6.js

```diff
@@ -42,22 +42,31 @@ function renderHero (site) {
   })}`
 }
 
-function renderTrackerDetails (companyListMap, DOMAIN_MAPPINGS) {
+function renderTrackerDetails (model, DOMAIN_MAPPINGS) {
+  const companyListMap = model.companyListMap || {}
   if (companyListMap.length === 0) {
     return bel`<li class="is-empty">None</li>`
   }
   if (companyListMap && companyListMap.length > 0) {
     return companyListMap.map((c, i) => {
-      if (c.name && c.name === 'unknown') c.name = '(Tracker network unknown)'
-      return bel`<li>
+      let borderClass = ''
+      if (c.name && c.name === 'unknown') {
+        c.name = '(Tracker network unknown)'
+      } else if (c.name && model.hasUnblockedTrackers(c)) {
+        const additionalText = ' associated domains'
+        const domain = model.site ? model.site.domain : c.name
```

Comment:
Showing the site domain instead of the company name for the unblocked paragraph, as requested in the task. The relationship between the site domain and the owner companies has yet to be explored for this scenario.
However, it's ok to show the company name as usual when we're blocking third party trackers.
Please feel free to reach out if this is unclear.